scripts/which-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+COMMIT=$1
+if [[ -z "${COMMIT}" ]]; then
+	echo "Usage: $0 <commit-ref>"
+	echo ""
+	echo -n "Example: $0 "
+	echo $'$(gh pr view <pr-num> --json mergeCommit | jq \'.mergeCommit.oid\' -r)'
+	exit 2
```

Comment:
You should source lib.sh like other scripts in this dir and use the `log` and `error` functions